client/src/pages/Login/LoginForm/useStyles.ts

```diff
@@ -6,14 +6,9 @@ const useStyles = makeStyles((theme) => ({
     marginTop: theme.spacing(1),
   },
   label: {
-    fontSize: '18px',
-    marginTop: '1rem',
-    fontWeight: 'bold',
-    textTransform: 'uppercase',
-  },
-  inputs: {
-    height: '2rem',
-    padding: '5px',
+    fontSize: 19,
+    color: 'rgb(0,0,0,0.4)',
```

Comment:
You are setting an opacity on the color however, `rgb` does not take an opacity, use `rgba` instead.

Reply:
Also, I am seeing, we are not using label class in `LoginForm`. I will remove it.